samples/msal-angular-v2-samples/angular10-sample-app/src/app/app.module.ts

```diff
@@ -25,10 +25,8 @@ export function loggerCallback(logLevel: LogLevel, message: string) {
 export function MSALInstanceFactory(): IPublicClientApplication {
   return new PublicClientApplication({
     auth: {
-      // clientId: '6226576d-37e9-49eb-b201-ec1eeb0029b6', // Prod enviroment. Uncomment to use. 
-      clientId: '3fba556e-5d4a-48e3-8e1a-fd57c12cb82e', // PPE testing environment
-      // authority: 'https://login.microsoftonline.com/common', // Prod environment. Uncomment to use.
-      authority: 'https://login.windows-ppe.net/common', // PPE testing environment.
+      clientId: 'ENTER_CLIENT_ID_HERE',
```

Comment:
What will the developer see if they don't update this value when they attempt to run the sample?

Reply:
msal will throw an endpoint_resolution error in this case. Good point though, I'll add a comment above that points to the registration instructions.

Reply:
done!

Reply:
Can we do a custom error in the sample to say their authority is not setup when `authority` matches `ENTER_CLIENT_ID_HERE`?  May be for a later PR, but `endpoint_resolution_error` is confusing and this is special case of folks running the sample as is. Thoughts??

Reply:
Sorry I missed this! Yes I think we could add a simple check before msal instantiation -I'm noting this down, best to tackle in another PR

Reply:
Works, I have already approved, so this should not block you.